runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -392,6 +414,16 @@ void setPoints(int points)
 		this.points = points;
 	}
 
+	public Instant getInfoTimer()
```

Comment:
generally you only need getters and setters when you try to access a field from outside the current class. When accessing the field in the current class you can just use `infoTimer  = Instant.now()` or `infoTimer != null`